index.html

```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
 <html lang="de">
   <head>
+    <link rel="stylesheet" href="./styles.css">
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Titel</title>
   </head>
   <body>
-    <h1 style="font-size:28px" style="font-family: Hello World">Hello World</h1>
-<p style="font-size:16px"style="color:blue">How are you doing, <p style="font-size:12px" style="" ;">I hop good</span>
+    <h1 >Hello World</h1>
+    <p>How are you doing,</p>
+    <p id="p2" >I hop good</p>
```

Comment:
```suggestion
    <p id="p2" >I hope good</p>
```

Hier fehlt ein 'e' ;)